app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -31,6 +31,9 @@ def parse
   end
 
   def pvm_instances
+    # collect all the placement group names to be consumed by each instance
+    collector.placement_groups_by_id
```

Comment:
Is this necessary?

Reply:
Yes

Reply:
:) haha. Let me expand on my question.

On line 126 you call `placement_group = collector.placement_group(instance.placement_group)`, which will call the `collector.placement_groups_by_id`. Since `placement_groups_by_id` is memoized it won't cost any extra API calls, but I don't think this line is necessary. Is there something I'm missing?

Reply:
Sure Jay. Let me recheck the code segment after removing the line.